### DISCUSSION TITLE
pyvips.error.Error: text does not support optional argument rgba

### DISCUSSION THREAD

**Discussion by soy-dev1** [machineonamission/mediaforge#136] [Q&A] [answered]
strange issue when trying to use $caption, I get this error and I have tried looking everywhere for documentation. Is this a bug or am I stupid?

```
DATETIME:2022-12-05 15:30:58.063418
COMMAND:$caption test
TRACEBACK:
Traceback (most recent call last):
  File "/root/.cache/pypoetry/virtualenvs/mediaforge-HXFskkj--py3.11/lib/python3.11/site-packages/discord/ext/commands/core.py", line 229, in wrapped
    ret = await coro(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mediaforge/src/commands/caption.py", line 115, in captioncommand
    await process(ctx, processing.vips.vipsutils.generic_caption_stack, [["VIDEO", "GIF", "IMAGE"]],
  File "/mediaforge/src/core/process.py", line 157, in process
    raise e
  File "/mediaforge/src/core/process.py", line 125, in process
    result = await run()
             ^^^^^^^^^^^
  File "/mediaforge/src/core/process.py", line 108, in run
    command_result = await func(*args, **kwargs)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mediaforge/src/processing/vips/vipsutils.py", line 21, in generic_caption_stack
    captext = await run_parallel(capfunc, *args, captions, size)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mediaforge/src/processing/common.py", line 130, in run_parallel
    raise res
pyvips.error.Error: text does not support optional argument rgba
```

**Comment by machineonamission** [comment]
Is this on a self-hosted copy? If so, make sure your libvips and pyvips are up to date

**Reply by soy-dev1** [reply]
it is self hosted, docker container. I have ran the update all and run option and it didn't fix it, should I run it on the actual host system?

**Reply by machineonamission** [reply]
Hmm that's weird, it should work. When did you install/create the docker container?

**Reply by soy-dev1** [reply]
two days ago, I have tried just about everything except for updating the VM, which I created around the same time. It is Ubuntu server if that helps. All packages should be up to date.

**Reply by machineonamission** [reply]
The host system shouldn't matter cause docker is basically a VM

I'll have a look at it and see what's causing this

**Reply by soy-dev1** [reply]
thank you

**Comment by machineonamission** [comment, marked as answer]
alright i found and fixed the issue. either rebuild the docker image from scratch or open the debug shell and run `apt-get -t testing --no-install-recommends install -y libvips-dev`. then run `exit` to get back to the menu

**Reply by soy-dev1** [reply]
works! thank you!
